packages/make-styles/src/renderer/createDOMRenderer.ts

```diff
@@ -24,52 +17,34 @@ export function createDOMRenderer(
 
     id: `d${lastIndex++}`,
 
-    insertDefinitions(dir, definitions): string {
-      let classes = '';
+    insertCSSRules(cssRules) {
       // eslint-disable-next-line guard-for-in
-      for (const propName in definitions) {
-        const definition = definitions[propName];
-        // 👆 [bucketName, className, css, rtlClassName?, rtlCSS?]
+      for (const styleBucketName in cssRules) {
+        const cssRulesForBucket = cssRules[styleBucketName as StyleBucketName]!;
+        const sheet = target && getStyleSheetForBucket(styleBucketName as StyleBucketName, target, renderer);
 
-        const className = definition[RULE_CLASSNAME_INDEX];
-        const rtlClassName = definition[RULE_RTL_CLASSNAME_INDEX];
+        for (let i = 0, l = cssRulesForBucket.length; i < l; i++) {
```

Comment:
if the `l` is an optimization it would deserve a comment

Reply:
```suggestion
        // This is a hot path in rendering styles: ".length" is cached in "l" var to avoid accesses the property
        for (let i = 0, l = cssRulesForBucket.length; i < l; i++) {
```

WDYT?

Reply:
Love it